tests/content/viewing-activity-parser-test.js

```diff
@@ -8,36 +9,36 @@ describe('ViewingActivityParser', function() {
 
   describe('parse', function() {
     it('creates the activity when synced_at is undefined', function() {
-      var activity = ViewingActivityParser.parse(new Date(undefined), window.movieActivity);
+      var activity = ViewingActivityParser.parse(moment(undefined, "YYYY-MM-DDTHH:mm:ss.SSSSZ"), window.movieActivity);
       expect(activity instanceof ViewingActivity).toBeTruthy();
     });
 
     it('creates the activity when synced_at equals to item date', function() {
-      var activity = ViewingActivityParser.parse(new Date('10/2/2015'), window.episodeActivity);
+      var activity = ViewingActivityParser.parse(moment('10/2/2015', 'M/D/YYYY'), window.episodeActivity);
       expect(activity instanceof ViewingActivity).toBeTruthy();
```

Comment:
@MrMamen Just ran this test here, it is failing because the date passed is before the `syncedAt`. This [condition](https://github.com/tegon/traktflix/pull/20/files#diff-fd41d2ff17a8b49f44d8e8c26de6084bR12) is returning `false` so the activity will be `undefined` failing the expect in the next line. If you change the first argument here to `moment('10/1/2015', 'M/D/YYYY')` it will pass.